services/app/apps/codebattle/assets/js/widgets/pages/builder/InputSignatureEditPanel.jsx

```diff
@@ -52,6 +52,10 @@ function InputSignatureEditPanel({
       return [true, ''];
     }
 
+    if (!Object.prototype.hasOwnProperty.call(suggest, 'argumentName')) {
+      return [false, ''];
+    }
```

Comment:
Сложносочиненная конструкция
Легче будет просто вытащить свойство

```suggestion
    if (suggest?.argumentName !== undefined)) {
      return [false, ''];
    }
```